azurerm/internal/services/cognitive/cognitive_account_resource.go

```diff
@@ -104,6 +108,83 @@ func resourceCognitiveAccount() *pluginsdk.Resource {
 				}, false),
 			},
 
+			"aad_client_id": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringIsNotEmpty,
+			},
+
+			"aad_tenant_id": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringIsNotEmpty,
+			},
+
+			"custom_subdomain_name": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringIsNotEmpty,
+			},
+
+			"disable_local_auth": {
```

Comment:
could we change this to be more consistant with the provider
```suggestion
			"local_auth_enabled": {
```